app/controllers/admin/sdg/targets_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::SDG::TargetsController < Admin::BaseController
+  Target = ::SDG::Target
+
+  def index
+    @targets = Target.all
```

Comment:
I forgot to add an `order` clause in the `Goal` model. We should add it here as well.

I've tried:

```ruby
Target.joins(:goal).merge(Goal.order(:code)).order("code")
```

But "1.2" is shown before "1.10" :relieved:.

Reply:
It seems that the sorting of alphanumeric strings is a difficult thing to achieve through both Ruby and SQL. I finally used Ruby to convert targets codes into numbers after a little bit of magic.

Reply:
Finally solved adding Comparable module to Target model.